samples/gke/simple/README.md

```diff
@@ -0,0 +1,38 @@
+This provides a sample for creating a simple AI cluster using [GKE](https://cloud.google.com/kubernetes-engine). This cluster uses:
```

Comment:
idk what the "correct" place for this info is but I have had trouble using the gke module due to iam permissions -- not actual problems with the module itself. Do you know what non-default permissions a user would need in order to run this?